gbbs/benchmarks/BatchDynamicConnectivity/SkipList/SkipList.h

```diff
@@ -18,7 +18,10 @@ struct SkipList {
 
         using pointers = std::pair<SkipListElement*, SkipListElement*>;
         using height_array = sequence<pointers>;
-        using values_array = sequence<sequence<sequence<std::pair<uintE, uintE>>>>;
+        using edge_type = std::pair<uintE, uintE>;
+        using values_one_side = sequence<edge_type>;
```

Comment:
This identifier is not very clear. What is a "side"? Can you choose a more informative name?